### PR TITLE
GamePlaying UI

### PR DESCRIPTION
**Pull request by taan02991**
- Also added bootstrap

### CODE REVIEW

File: vue/src/router/index.js [hunk 1]
```diff
@@ -9,6 +10,10 @@ const routes = [
     path: "/",
     component: Index,
   },
+  {
+    path: "/game-playing/:id",
+    component: GamePlaying,
+  },
```
Comment (UtopiaBeam) [inline vue/src/router/index.js]: This route should use dynamic import to enable lazy-loading (More detail: https://router.vuejs.org/guide/advanced/lazy-loading.html).

However, I'm not sure if it works on starport (It should work right?)

**Review by UtopiaBeam — COMMENTED** [review]
I really want to avoid using bootstrap because it uses jQuery. Maybe use tailwindcss instead.

**Review by UtopiaBeam — CHANGES_REQUESTED** [review]
Please run linter.

File: vue/src/router/index.js (hunk 1 above)
Comment (taan02991) [reply vue/src/router/index.js]: Yes, it worked

**Comment by taan02991** [comment]
> I really want to avoid using bootstrap because it uses jQuery. Maybe use tailwindcss instead.

I looked at tailwindcss docs and I think it's ok. I will try it.

**Review by UtopiaBeam — APPROVED** [review]

**Pull request merged**